### PR TITLE
[duplicati] Move duplicati to Common-Chart

### PR DESCRIPTION
**Pull request by Ornias1993**
**Description of the change**

This should be most of the work to port Duplicati to the Common-Chart.

**Benefits**

Easier maintanance

**Possible drawbacks**

Might break something, though it shouldn't

**Additional information**

Need to be checked by the maintainer

TODO:
Need to (re)generate the readme, don't have a linux dev env running atm

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/duplicati/values.yaml [hunk 1]
```diff
@@ -10,99 +10,31 @@ image:
...
-    ## If you want to reuse an existing claim, you can pass the name of the PVC using
-    ## the existingClaim variable
-    # existingClaim: your-claim
-    accessMode: ReadWriteOnce
-    size: 1Gi
-    ## Do not delete the pvc upon helm uninstall
-    skipuninstall: false
+    enabled: false
+    emptyDir: false
+    mountPath: /config
```
Comment (rolfberkenbosch) [inline charts/duplicati/values.yaml:29]:
```suggestion
    mountPath: /config
    accessMode: ReadWriteOnce
    size: 1Gi
```

File: charts/duplicati/values.yaml [hunk 2]
```diff
@@ -10,99 +10,31 @@ image:
...
-    ## If you want to reuse an existing claim, you can pass the name of the PVC using
-    ## the existingClaim variable
-    # existingClaim: your-claim
-    accessMode: ReadWriteOnce
-    size: 1Gi
-    ## Do not delete the pvc upon helm uninstall
-    skipuninstall: false
+    enabled: false
+    emptyDir: false
+    mountPath: /source
```
Comment (rolfberkenbosch) [inline charts/duplicati/values.yaml:33]:
```suggestion
    mountPath: /source
    accessMode: ReadWriteOnce
    size: 1Gi
```

File: charts/duplicati/values.yaml [hunk 3]
```diff
@@ -10,99 +10,31 @@ image:
...
   backups:
-    enabled: true
+    enabled: false
+    emptyDir: false
+    mountPath: /backups
```
Comment (rolfberkenbosch) [inline charts/duplicati/values.yaml:37]:
```suggestion
    mountPath: /backups
    accessMode: ReadWriteOnce
    size: 1Gi
```

**Review by rolfberkenbosch — CHANGES_REQUESTED** [review]
If you add my changes to the code, then it will works also with the PVC.
I have just test the code with my changes, and it works!

**Comment by Ornias1993** [comment]
@rolfberkenbosch I'm not going to finish this (i've more important projects that need my attention currently), it's just a PoC to make clear it has nothing to do with refactoring, like you suggested.

You can copy it and finished it if you like.

Two notes:
- RWO is already the default so doesn't have to be added afaik
- specifying a size is not needed at all and is normally not done on all the other common-dependent charts

**Comment by bjw-s** [comment]
If you never meant to work on this (for it to be actually merged), why did you create a PR for it? Was it really just to prove a point and "prove" that you are right? You could have just linked to your branch instead of creating this, and even that is debatable.

We have asked you a few times before to remain cordial and be kind to the people that are contributing to our community. Please consider this your final explicit warning.

**Comment by Ornias1993** [comment]
> If you never meant to work on this (for it to be actually merged), why did you create a PR for it? Was it really just to prove a point and "prove" that you are right? You could have just linked to your branch instead of creating this, and even that is debatable.
> 
> We have asked you a few times before to remain cordial and be kind to the people that are contributing to our community. Please consider this your final explicit warning.

Its called “helping out” AND proving a point. It never was a full blown PR, it always was a draft aka PoC.

I was never planning **not** to finish it btw, I never said so. Other things came up that are more important than spinning up a vm/container just to generate a wiki. 

Maybe before warning people, try and look at the intend of people FIRST.

**Comment by onedr0p** [comment]
> To be clear: i’m always direct, if I see BS, I call it BS. Though I prefer to cleanup along the wag while at it, like this draft.

We want this community to be friendly and helpful. We're all not geniuses and sometimes we can say things that could be wrong or could be construed as bullshit.

The way you comment or provide feedback on issues is often unacceptable to us and are viewed as hostile behavior.

Please work on your communication skills.

**Comment by onedr0p** [comment]
@all-contributors please add @Ornias1993 for code

**Pull request closed without merging**